src/index.web.ts

```diff
@@ -0,0 +1,54 @@
+export const MMKV = {
+  /**
+   * Set a value for the given `key`.
+   */
+  set: (key: string, value: boolean | string | number): void => {
+    localStorage.setItem(key, value.toString());
+  },
+  /**
+   * Get a boolean value for the given `key`.
+   *
+   * @default false
+   */
+  getBoolean: (key: string): boolean => {
+    let result = localStorage.getItem(key);
+
+    return result ? Boolean(result) : false;
+  },
+  /**
+   * Get a string value for the given `key`.
+   *
+   * @default undefined
+   */
+  getString: (key: string): string | undefined => {
+    let result = localStorage.getItem(key);
+
+    return result != null ? result : undefined;
```

Comment:
In TypeScript (and modern JavaScript) this can be written as:

```suggestion
    return result ?? undefined;
```

Reply:
Thanks, I did not realize that `??` does not follow standard JS truthy and instead only null/undefined.